mayastor/src/bdev/dev/aio.rs

```diff
@@ -99,10 +98,17 @@ impl CreateDestroy for Aio {
                     name: self.get_name(),
                 })
                 .map(|name| {
-                    self.uuid.map(|u| {
-                        Bdev::lookup_by_name(&name)
-                            .map(|mut b| b.set_uuid(Some(u.to_string())))
-                    });
+                    if let Some(mut bdev) = Bdev::lookup_by_name(&self.name) {
+                        if let Some(uuid) = self.uuid {
+                            bdev.set_uuid(Some(uuid.to_string()));
+                        }
+                        if !bdev.add_aliases(&self.aliases) {
```

Comment:
we can simply call bdev.add_alias(&url.to_string()) and avoid the allocation of the Vec etc, right?

Reply:
yes that's right, the idea was to make it ready to support multiple aliases, should we want to but yes not really necessary for now...